File: src/main/java/com/andrepixel/backend_blog/BackendBlogApplication.java
package com.andrepixel.backend_blog;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.cloud.openfeign.FeignAutoConfiguration;

@SpringBootApplication
@EnableFeignClients
@ImportAutoConfiguration({FeignAutoConfiguration.class})
public class BackendBlogApplication {

    public static void main(String[] args) {
        SpringApplication.run(BackendBlogApplication.class, args);
    }

}

File: src/main/kotlin/com/andrepixel/backend_blog/BackendBlogApplication.kt
package com.andrepixel.backend_blog

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class BackendBlogApplication {
	fun main(args: Array<String>) {
		runApplication<BackendBlogApplication>(*args)
	}
}